src/main/java/bookstore/model/Book.java

```diff
@@ -5,11 +5,13 @@
 import jakarta.persistence.GeneratedValue;
 import jakarta.persistence.GenerationType;
 import jakarta.persistence.Id;
+import jakarta.persistence.Table;
 import java.math.BigDecimal;
 import lombok.Data;
 
 @Entity
 @Data
```

Comment:
better do not use @Data with Entity class
@Getter
@Setter
@NoArgsConstructor - is good enough and you will control equals/hashCode/toString